lib/rules/id-length.js

```diff
@@ -117,6 +117,8 @@ module.exports = {
                 return properties && !parent.computed && parent.key.name === node.name;
             },
             ImportDefaultSpecifier: true,
+            ImportSpecifier: true,
+            ImportNamespaceSpecifier: true,
             RestElement: true,
```

Comment:
```suggestion
            ImportDefaultSpecifier: true,
            ImportNamespaceSpecifier: true,
            RestElement: true,
```

as proposed in the issue we only gonna report `ImportNamespaceSpecifier`